runner_manager/jobs/runner.py

```diff
@@ -0,0 +1,9 @@
+from runner_manager import RunnerGroup
+from runner_manager.clients.github import GitHub
+from runner_manager.dependencies import get_github
+
+
+def create_runner(group: RunnerGroup) -> str | None:
+    github: GitHub = get_github()
+    runner = group.create_runner(github)
+    return runner.pk if runner is not None else runner
```

Comment:
`else runner` means returning None. We can write better then.

```suggestion
    if runner is not None:
        return runner.pk
```